playbooks/byo/openshift_facts.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- name: Gather OpenShift facts
+- name: Gather Cluster facts
```

Comment:
:+1: love the use of Cluster here...  Gets the point across while still being generic.